app/jobs/upvs/find_public_authority_edesk_list_job.rb

```diff
@@ -0,0 +1,18 @@
+require 'harvester_utils/downloader'
+
+class Upvs::FindPublicAuthorityEdeskListJob < ApplicationJob
+  queue_as :upvs
+
+  DATASET_URL = 'https://data.gov.sk/dataset/upvs-institucie-so-schrankou'
+
+  def perform(downloader: HarvesterUtils::Downloader, fetch_job: Upvs::FetchPublicAuthorityEdesksListJob)
+    html = downloader.download(DATASET_URL)
+    doc = Nokogiri::HTML.parse(html)
+    doc.search('.resource-item .dropdown-menu a').each do |a|
+      if a['href'].include?('.csv')
+        fetch_job.perform_later(a['href'])
+        break
+      end
```

Comment:
Sem by som dal nejaku exception pre pripad, ze ten link nenajdeme. Aby to nefailovalo potichu.